website_docs/context_propagation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Ruby Context Propagation"
+weight: 24
+description: >
+  <img width="35" src="https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/32x32/Ruby_SDK.svg"></img>
+  A language-specific implementation of OpenTelemetry in Ruby.
+---
+
+## Context Propagation
+
+Traces can extend beyond a single process. This requires _context propagation_, a mechanism where identifiers for a trace are sent to remote processes.
+
+In order to propagate trace context over the wire, a propagator must be registered with the OpenTelemetry SDK.
+The W3 TraceContext and Baggage propagators are configured by default, however operators may override this value by setting `OTEL_PROPAGATORS` environment variable to a comma separated list of [propagators](https://github.com/open-telemetry/opentelemetry-ruby/tree/main/propagator).
+
+```sh
+export OTEL_PROPAGATORS=tracecontext,baggage,b3
+```
+
+You will also have to add a gem dependency:
+
+```sh
+gem install opentelemetry-propagator-b3
```

Comment:
At this point in the Ruby ecosystem, could we assume that most people are using Bundler and Gemfiles? Can we streamline the setup instructions a bit with only the "add gem to Gemfile and bundle install" path and assume that people who are not using Bundler know to do a gem install instead? My impression is that most people new to Ruby are learning the Bundler Way and `gem install` is choice made by more experienced folks, but my impression could certainly be mistaken!

Reply:
I agree with you. Let's see what others have to say about it.

Reply:
I'm +1 on preferring `bundler` instructions rather than `gem` instructions.